### ISSUE TITLE
[Legitimate Site Blocked] casino.anitamaxwynn.biz

### ISSUE THREAD

**Issue by slamtoken** [MetaMask/eth-phishing-detect#36601]
https://casino.anitamaxwynn.biz/originals

**Comment by AlexHerman1** [comment]
can you please provide any further information or context regarding this site? Is the team doxxed? is there a social media presence for the project?

**Comment by slamtoken** [comment]
Yes, of course, this is a token with an active community

See all the links here: https://linktr.ee/anitamaxwynntoken
This is a casino site using a web3 login to create an account. It's a whitelabel version of panda.co, which is using the same backend.

<img width="401" alt="Screenshot 2024-04-02 at 16 02 23" src="https://github.com/MetaMask/eth-phishing-detect/assets/83141278/50fd4e5a-04e8-474c-b11a-034d6dc04c40">

Thank you. 

PS. I have opened another issue regarding one of our other domains #36612

**Comment by AlexHerman1** [comment]
unblocked both domains
